src/pages/ReimbursementAccount/ConnectBankAccount/ConnectBankAccount.tsx

```diff
@@ -11,37 +10,31 @@ import useLocalize from '@hooks/useLocalize';
 import useThemeStyles from '@hooks/useThemeStyles';
 import BankAccount from '@libs/models/BankAccount';
 import EnableBankAccount from '@pages/ReimbursementAccount/EnableBankAccount/EnableBankAccount';
-import * as ReimbursementAccountProps from '@pages/ReimbursementAccount/reimbursementAccountPropTypes';
 import * as Report from '@userActions/Report';
 import ONYXKEYS from '@src/ONYXKEYS';
+import type {Account, ReimbursementAccount} from '@src/types/onyx';
 import BankAccountValidationForm from './components/BankAccountValidationForm';
 import FinishChatCard from './components/FinishChatCard';
 
-const propTypes = {
-    /** Bank account currently in setup */
-    reimbursementAccount: ReimbursementAccountProps.reimbursementAccountPropTypes.isRequired,
-
-    /** Handles back button press */
-    onBackButtonPress: PropTypes.func.isRequired,
-
+type ConnectBankAccountOnyxProps = {
     /** User's account who is setting up bank account */
-    account: PropTypes.shape({
-        /** If user has two-factor authentication enabled */
-        requiresTwoFactorAuth: PropTypes.bool,
-    }),
+    account: OnyxEntry<Account>;
 };
 
-const defaultProps = {
-    account: {
-        requiresTwoFactorAuth: false,
-    },
+type ConnectBankAccountProps = ConnectBankAccountOnyxProps & {
+    /** Bank account currently in setup */
+    reimbursementAccount: ReimbursementAccount;
+
+    /** Handles back button press */
+    onBackButtonPress: () => void;
 };
 
-function ConnectBankAccount({reimbursementAccount, onBackButtonPress, account}) {
+function ConnectBankAccount({reimbursementAccount, onBackButtonPress, account}: ConnectBankAccountProps) {
     const styles = useThemeStyles();
     const {translate} = useLocalize();
 
-    const bankAccountState = lodashGet(reimbursementAccount, 'achData.state', '');
+    const handleNavigateToConciergeChat = () => Report.navigateToConciergeChat();
```

Comment:
No need to create new method on every render.

```suggestion
    const handleNavigateToConciergeChat = Report.navigateToConciergeChat;
```

Reply:
in this case is needed because `Report.navigateToConciergeCha` have different params than onPress callback
![image](https://github.com/Expensify/App/assets/7682108/1af36360-42d0-4185-b5fc-5540626ad7b2)

Reply:
ok, fair enough 👍